packages/web3-providers-ipc/package.json

```diff
@@ -22,7 +22,7 @@
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
 		"test:watch": "npm test -- --watch",
 		"test:unit": "jest --config=./test/unit/jest.config.js",
-		"test:integration": "jest --config=./test/integration/jest.config.js --passWithNoTests"
+		"test:integration": "jest --config=./test/integration/jest.config.js --passWithNoTests --forceExit"
```

Comment:
Is `--forceExit` on for debugging?

Reply:
Locally it is not stuck, but in GitHub Action, sometimes yes. I need to investigate it deeply

Reply:
If its not fixed in current PR pls create issue pointing to this discussion.

Reply:
I've fixed the test. So it works without `--forceExit`